scripts/setup_script_e2e.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+shout() {
+  set +x
+  echo -e "\n!!!!!!!!!!!!!!!!!!!!\n${1}\n!!!!!!!!!!!!!!!!!!!!\n"
+  set -x
+}
+
+set -ex
+
+shout "Setting up some stuff"
+
+# Create a bin directory whereever script runs. This will be where all binaries that need to be in PATH will reside.
+mkdir bin artifacts
+# Change the default location of go's bin directory (without affecting GOPATH). This is where compiled binaries will end up by default
+# for eg go get ginkgo later on will produce ginkgo binary in GOBIN
+export GOBIN="`pwd`/bin"
+export GOBIN_TEMP=$GOBIN
+# Set kubeconfig to current dir. This ensures no clashes with other test runs
+export KUBECONFIG="`pwd`/config"
+export ARTIFACTS_DIR="`pwd`/artifacts"
+export CUSTOM_HOMEDIR=$ARTIFACTS_DIR
```

Comment:
These are linux paths which are used as env values https://github.com/openshift/odo/blob/ba7161940c735caffaa9bd49e731dd15b8e9a61f/pkg/testingutil/configs.go#L19 Thus those won't work in case of windows.

Reply:
Added, Thanks @mik-dass